HorizonApache/src/main/java/com/hubspot/horizon/apache/ApacheHttpClient.java

```diff
@@ -81,6 +81,10 @@ public ApacheHttpClient(HttpConfig config) {
     this.defaultOptions = config.getOptions();
     this.timer = new Timer("http-request-timeout", true);
   }
+  
+  public boolean usesSocksProxy() {
```

Comment:
Should we call this `isSocksProxied` to match the delegate?

Reply:
Yes, good point